locales/fr.lua

```diff
@@ -0,0 +1,22 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul",
+        allowed_ext = "extensions sont autorisés.",
+
```

Comment:
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul",
```
`allowed_ext` is not used.